test/logger_test.rb

```diff
@@ -0,0 +1,75 @@
+require 'test_helper'
+
+describe Lotus::Logger do
+
+  before do
+    #clear defined class
+    Object.send(:remove_const, :TestLogger) if Object.constants.include?(:TestLogger)
+  end
+
+  it 'like std logger, default log level is info' do
+    class TestLogger < Lotus::Logger; end
+    assert_respond_to(TestLogger.new, :info?)
```

Comment:
This doesn't test if `info?` returns true, it test if it responds to the method.
What about `assert TestLogger.new.info?, "failure message"`.